ui/app/src/components/FormsEngine/components/CreateModeHeader.tsx

```
@@ -44,7 +45,9 @@ export function CreateModeHeader({ path }: { path: string }) {
 					}}
 				/>
 			</Typography>
-			<Typography color="textSecondary" variant="body2" children={path} />
+			<Collapse in={!collapse}>
+				<Typography color="textSecondary" variant="body2" children={path} />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)